zstacklib/zstacklib/test/lvm/test_safe_write_file.py

```
@@ -0,0 +1,40 @@
+
+            linux.safe_write_file(os.path.join(tmp_path, "d1/f1"), content="test_data2")
+            self.assertEqual(linux.read_file(os.path.join(tmp_path, "d1/f1")), "test_data2")
+            st = os.stat(os.path.join(tmp_path, "d1/f1"))
+            self.assertEqual(stat.S_IMODE(st.st_mode), 0o666)
+
+            linux.safe_write_file(os.path.join(tmp_path, "d1/f1"), content="test_data3", keep_origin_attrs=False)
+            self.assertEqual(linux.read_file(os.path.join(tmp_path, "d1/f1")), "test_data3")
+            st = os.stat(os.path.join(tmp_path, "d1/f1"))
+            self.assertEqual(stat.S_IMODE(st.st_mode), 0o644)
+
+            with self.assertRaises(Exception):
```

Ruff (0.14.14)

Do not assert blind exception: `Exception`

(B017)